smoke/diagnostics/a.rbs

```diff
@@ -4,6 +4,10 @@ end
 
 class MethodDefinitionMissing
   def foo: () -> void
+
+  attr_reader    bar: untyped
+  attr_writer    baz: untyped
+  attr_accessor  qux: untyped
```

Comment:
Without the fix, reproduces the issue. With the fix it passes by not having errors in `test_expectations.yml`.

Should this be anywhere else instead?